pandas/core/accessor.py

```diff
@@ -116,13 +116,15 @@ def _setter(self, new_values):
                 doc=getattr(delegate, accessor_mapping(name)).__doc__,
             )
 
+        import functools as ft
```

Comment:
Very elegant solution for the signatures. Just curious, why not put the import at the top of the file？

Reply:
Felt it would help with readability this way, have seen similar imports elsewhere as well.

Reply:
sure